### DISCUSSION TITLE
Change Summaries and Element Aspects

### DISCUSSION THREAD

**Discussion by nn15296** [iTwin/itwinjs-core#2050] [Q&A]
Hi, I'm using the below query to find the elements which have been updated in a given Change Summary. Does it still work if there have been no updates to the element's direct properties, but the element's aspects have been updated? Or would you need to run the query for all the aspect classes and link them to their associated elements?

Thanks

![image](https://user-images.githubusercontent.com/33571096/129234283-cdb50e35-98de-41ea-b895-c84b2fb0091d.png)

**Comment by diegopinate** [comment]
You are correct. You would need to query the aspects directly. The change summary will only tell you which instances have changes. In the case that only the aspect is changed, and not the element, only the aspect will be in the changeset and thus will be the only thing showing in the change summary. @khanaffan
